bin/extract-abi

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+/**
+ * Extract the pure ABI from the contracts.json and pipe it to stdout
+ * Usage: extract-abi build/contracts/mycontract.json > contractabi.json
+ */
+const fs = require('fs')
+
+const filepath = process.argv[2]
+if (!filepath || !fs.statSync(filepath)) throw new Error(`File does not exist: ${filepath}`)
+
+const json = JSON.parse(fs.readFileSync(process.argv[2], 'utf8'))
+
+process.stdout.write(Buffer.from(JSON.stringify(json.abi)) + '\n')
```

Comment:
would `console.log(JSON.stringify(json.abi))` accomplish the same here?

Reply:
yes

Reply:
not 💯 sure how `console.log` and `process.stdout` behave different when piping though. `stdout` feels like the safer choice to work with pipes etc

Reply:
![image](https://user-images.githubusercontent.com/1504952/29987900-bdd498e2-8f6a-11e7-8e2e-ad3ef793e5ab.png)

lets go for `console.log` then, seeing `\n` and `Buffer` and `stdout` makes me not feeling node-like ;)